core/DCache.hpp

```diff
@@ -23,6 +23,7 @@ namespace olympia {
             PARAMETER(uint32_t, l1_associativity, 8, "DL1 associativity (power of 2)")
             PARAMETER(uint32_t, cache_latency, 1, "Assumed latency of the memory system")
             PARAMETER(bool, l1_always_hit, false, "DL1 will always hit")
+            PARAMETER(bool, l1_l2cache_credit_available, true, "Aloowing L1 cache to send the first miss request to L2Cache")
```

Comment:
typo for `Allowing`

Reply:
What happens if this is false? When will Dcache be able to send miss request to L2Cache?

Reply:
This changed to a uint32_t type now. It will be programmed by L2Cache into DCache through the *ack_ port at the beginning of the simulation. This represents the number of l2cache_credits for DCache.

At every de-allocation from the input_buffer, L2Cache will update DCache with new number of empty slots in the input_buffer.